lib/rules/jsx-no-target-blank.js

```diff
@@ -103,7 +104,47 @@ module.exports = {
           context.report({
             node,
             message: 'Using target="_blank" without rel="noreferrer" '
-              + 'is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener'
+              + 'is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener',
+            fix(fixer) {
+              const relAttribute = node.parent.attributes.find((attr) => attr.name.name === 'rel');
+              if (!relAttribute) {
+                return fixer.insertTextAfter(node, ' rel="noreferrer"');
+              }
+
+              if (!relAttribute.value) {
+                return fixer.insertTextAfter(relAttribute, '="noreferrer"');
+              }
+
+              if (relAttribute.value.type === 'Literal') {
+                if (!relAttribute.value.value.trim()) {
+                  return fixer.replaceText(relAttribute.value, '"noreferrer"');
+                }
+
+                const parts = relAttribute.value.value
+                  .split('noreferrer')
+                  .filter((part) => part);
+                return fixer.replaceText(relAttribute.value, `"${parts.join(' ')} noreferrer"`);
+              }
+
+              if (relAttribute.value.expression.type === 'Literal') {
+                const valueType = typeof relAttribute.value.expression.value;
+                if (
+                  ['boolean', 'number', 'symbol', 'bigint', 'undefined'].includes(valueType)
```

Comment:
what about `null`?

Reply:
That is caught by the `|| <expr>` on line 133 (right below this one). Remember, `typeof null` results in `"object"`.